bridge_adaptivity/module/views.py

```diff
@@ -406,25 +407,36 @@ def post(self, request, *args, **kwargs):
         Updating activity and changing the activity's order if activity changes the type.
         """
         activity = self.get_object()
-        # NOTE(AndreyLykhoman): 'is_change_type' is an indicator of the type change in the activity. If the
-        #  'is_change_type' is True, we will have to change the activity's order.
-        is_change_atype = request.POST.get("atype") != activity.atype
-        result = super().post(request, *args, **kwargs)
-        if is_change_atype:
-            # NOTE(AndreyLykhoman): Getting updated activity.
-            activity = self.get_object()
-            ordering_queryset = activity.get_ordering_queryset()
-            ordering_queryset = ordering_queryset.exclude(pk=activity.pk)
-            # NOTE(AndreyLykhoman): if the ordering_queryset contains one or more activities, we can get last element's
-            #  index, increase by one and set to updating activity's order. If the ordering_queryset is empty, we need
-            #  to set zero to updating activity's order
+        # NOTE(AndreyLykhoman): Run this block if activity changes it's 'atype'. In this block We reorder set where
```

Comment:
why do word `We` is started with the capital letter? Whom you are talking about?